src/role-access/components/case-roles-table/case-roles-table.component.ts

```diff
@@ -33,4 +33,8 @@ export class CaseRolesTableComponent {
     this.items = value.map((role) => ({...role, open: false}));
     this._roles = value;
   }
+
+  public getActionHref(action: any, item: Item): string {
```

Comment:
I think we can set an explicit type  `action: Action` instead of `any`

Reply:
Agree

Reply:
Done